src/vars/lists/assoc_lists.md

```diff
@@ -1,14 +1,14 @@
 # Associated lists
 
-## ⚠⚠ WIP PAGE ⚠⚠
-
-Associated lists, or list associations, add a unique functionality to regular lists by allowing you to associate values within the list with another value often referred to in other languages as a `key-value pair` or dictionary. This can be done as such:
+Associated lists, or list associations, add a unique functionality to regular lists by allowing you to associate values within the list with another value often referred to in other languages as a `key-value pair`, `map` or `dictionary`. This can be done as such:
```

Comment:
```suggestion
Associated lists, or list associations, add a unique functionality to regular lists by allowing you to associate values within the list with another value often referred to in other languages as a `key-value pair`, `map`, or `dictionary`. This can be done as such:
```